examples/istio-with-separate-cert/files/bigbangdev.cert

```diff
@@ -0,0 +1,91 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Why are we committing these to git if they are also loaded via a maefile?  I think it should be one or the other.

Reply:
It would require adding a dependency on [yq](https://github.com/mikefarah/yq) for the user to be able to perform this example. Doing it this way keeps the dependency list the same as it was. I added a helper make target for maintainers to quickly and easily update the cert.

FWIW, the Big Bang example also has the cert in the codebase, by having it in the `values.yaml` file.